ovos_PHAL_plugin_commonIOT/device_manager.py

```diff
@@ -34,6 +34,104 @@ def __init__(self, bus):
         # self.bus.on("ovos.iot.device.volume.down", self.handle_volume_down)
         # self.bus.on("ovos.iot.device.mute", self.handle_mute)
         # self.bus.on("ovos.iot.device.unmute", self.handle_unmute)
+        
+        # iot lights actions
+        self.bus.on("ovos.iot.device.get.brightness", self.handle_get_brightness)
+        self.bus.on("ovos.iot.device.set.brightness", self.handle_set_brightness)
+        self.bus.on("ovos.iot.device.decrease.brightness", self.handle_decrease_brightness)
+        self.bus.on("ovos.iot.device.increase.brightness", self.handle_increase_brightness)
+        self.bus.on("ovos.iot.device.get.color", self.handle_get_color)
+        self.bus.on("ovos.iot.device.set.color", self.handle_set_color)
+        
+    # light handlers 
+    def handle_get_brightness(self, message):
+        device = self.get_device(message.data.get("device"))
+        if not device:
+            self.speak_dialog("device.not.found.dialog", data="device": message.data.get("device").get("name"))
+        else:
+            brightness = device.brightness
+            self.speak_dialog("light.current.brightness.dialog", data={"device": device.name,
+                                                                       "brightness": brightness})
+            return brightness
+        return False
+    
+    def handle_set_brightness(self, message):
+        device = self.get_device(message.data.get("device"))
+        if not device:
+            self.speak_dialog("device.not.found.dialog", data="device": message.data.get("device").get("name"))
+        else:
+            try:
+                brightness = message.data.get("brightness")
+                device.set_brightness(brightness)
+                self.speak_dialog("light.set.brightness.dialog", data={"device": device.name,
+                                                                       "brightness": brightness})
+                return device.brightness
+            
+            except Exception as e:
+                LOG.error(e)
+                self.speak_dialog("light.set.brightness.error.dialog", data={"device": device.name})
```

Comment:
dialogs should all be in the vui.py file, this should only have a bus api and interact with device in device_manager.py